src/server/views/lists/partials/lawyers/lawyers-results-list.njk

```diff
@@ -11,7 +11,6 @@
 {% if searchResults.length === 0 %}
   <p class="govuk-body">We couldn’t find any results that match your search terms. Try changing these to search again.</p>
 {% else %}
-  <h2 class="govuk-heading-l govuk-!-margin-bottom-4">Lawyers in {{ country }}</h2>
```

Comment:
Is this text moved to another part of the project?

Reply:
Yep (and should have been h1!).

see `/find/:serviceType/results.njk`

```
{% extends "./../../results-page.njk" %} // "Use the results-page.njk layout" 

{% block heading %} // blocks are like "slots" in svelte or react for example. "In the results-page slot for "heading", put this in it:
  <h1 class="govuk-heading-l">
    Funeral directors {{ "in " + _.startCase(country) if country else "abroad"}}
  </h1>
{% endblock %}
```